zkvm/src/txlog.rs

```diff
@@ -67,15 +67,16 @@ impl MerkleItem for Entry {
                 t.commit_point(b"retire.q", q);
                 t.commit_point(b"retire.f", f);
             }
-            Entry::Input(utxo) => {
-                t.commit_bytes(b"input", &utxo.0);
+            Entry::Input(id) => {
+                t.commit_bytes(b"input", id.as_bytes());
             }
-            Entry::Nonce(pred, maxtime) => {
-                t.commit_point(b"nonce.p", &pred);
-                t.commit_u64(b"nonce.t", *maxtime);
+            Entry::Output(output) => {
```

Comment:
Why do we have `Entry::Output` contain the whole output structure if we only log the id? could we have it mirror input & only contain the id?

Reply:
Placing whole output there comes at no cost and it's useful in /token project where the output is taken out of the log w/o having to insert the hooks into VM execution to get it out for indexing.